### DISCUSSION TITLE
When RemoveRow use, cells are unmerged

### DISCUSSION THREAD

**Discussion by hiroto-nagaoka** [PHPOffice/PhpSpreadsheet#4116] [Q&A] [answered]
When you use removeRow in PhpSpreadsheet, the cells in the next row of the deleted row are unmerged.

**Comment by oleibman** [comment, marked as answer]
I am not able to duplicate your problem from your description. I tried the following program:
```php
        $spreadsheet = new Spreadsheet();
        $sheet = $spreadsheet->getActiveSheet();
        $sheet->getCell('A1')->setValue(1);
        $sheet->getCell('A2')->setValue('hello');
        $sheet->getCell('D2')->setValue(2);
        $mergeRange = 'A2:C2';
        $sheet->mergeCells($mergeRange);
        $sheet->removeRow(1);
        $writer = new XlsxWriter($spreadsheet);
        $outfile = 'issue.4116.xlsx';
        $writer->save($outfile);
        echo "saved $outfile\n";
```
When I open the saved spreadsheet, the first row has been removed, and the cells remain merged in the new first row.
![image](https://github.com/user-attachments/assets/6571fe29-96da-481a-95bf-f372577b2b47)

**Reply by hiroto-nagaoka** [reply]
It may be due to other processing.
I'll try it again.
thank you
